ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -288,10 +288,8 @@ final class SandboxServer(
         ledgerConfiguration = ledgerConfiguration,
         commandConfig = config.commandConfig,
         partyConfig = PartyConfiguration.default.copy(
-          // In this version of Sandbox, parties are always allocated implicitly. Enabling
-          // this would result in an extra `writeService.allocateParty` call, which is
-          // unnecessary and bad for performance.
-          implicitPartyAllocation = false,
+          // sandbox-classic always allocates party implicitly
+          implicitPartyAllocation = true,
```

Comment:
Previously `ActiveLedgerStateManager` did the implicit party allocation. But now that it's gone, we resort to the same mechanism as sadbonx (i.e. explicit implicit party allocation), right?

Reply:
Precisely, if this flag is on the command submission service walks the transaction tree and does the party allocation.